hospital/delivery/hospital_handler.go

```
@@ -0,0 +1,54 @@
+package delivery
+
+import (
+	"github.com/gin-gonic/gin"
+	"github.com/ribhavsharma/my-go-app/hospital"
+	"github.com/ribhavsharma/my-go-app/models"
+	"net/http"
+	"strconv"
+)
+
+type HospitalHandler struct {
+	hospitalUsecase hospital.Usecase
+}
+
+func (h HospitalHandler) Fetch(context *gin.Context) {
+	hospital, err := h.hospitalUsecase.Fetch()
+	if err !=nil{
+		panic(err)
+		}
+	context.JSON(http.StatusOK, hospital)
+}
+
+func (h HospitalHandler) GetByID(context *gin.Context) {
+	tempId, err := strconv.Atoi(context.Param("id"))
+	if err != nil{
+		panic(err)
+	}
+	id := int64(tempId)
+	hospital, err := h.hospitalUsecase.GetById(id)
+	context.JSON(http.StatusOK, hospital)
+
+}
+
+func (h HospitalHandler) NewHospital(context *gin.Context) {
+	hospital := models.Hospital{}
+	err := context.Bind(&hospital)
+	if err != nil{
+		context.JSON(http.StatusOK, hospital)
```

This should return an error, not a `StatusOK`